phidgets_current_inputs/CHANGELOG.rst

```diff
@@ -0,0 +1,21 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package phidgets_current_inputs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1.0.5 (2022-02-17)
+------------------
+
+1.0.4 (2021-10-22)
+------------------
+
+1.0.3 (2021-09-29)
+------------------
+
+1.0.2 (2021-03-09)
+------------------
+
+1.0.1 (2020-06-04)
+------------------
+
+1.0.0 (2020-06-03)
+------------------
```

Comment:
```suggestion
Forthcoming
-----------
```

Reply:
Actually, it is better if you don't add a CHANGELOG at all.  We have automated tools to do that.